test/integration/fleetscope/fleetscope_test.go

```diff
@@ -0,0 +1,39 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package fleetscope
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test/pkg/tft"
+)
+
+func TestFleetscope(t *testing.T) {
+
+	for _, envName := range []string{
+		"development",
+		"non-production",
+		"production",
+	} {
+		t.Run(envName, func(t *testing.T) {
+			fleetscope := tft.NewTFBlueprintTest(t,
+				tft.WithTFDir(fmt.Sprintf("../../../3-fleetscope/envs/%s", envName)),
```

Comment:
@yliaog - This will resolve the current INT blocker:

```suggestion
				tft.WithTFDir(fmt.Sprintf("../../../4-fleetscope/envs/%s", envName)),
```

Reply:
applied, thanks.